config/database.yml

```
@@ -1,7 +1,7 @@
 default: &default
   adapter: postgis
   encoding: unicode
-  url: <%= ENV["DATABASE_URL"] %>
+  url: <%= ENV.fetch('DATABASE_URL', '').sub(/^postgres/, 'postgis') %>
```

How about this:

```suggestion
  url: <%= ENV.fetch('DATABASE_URL', '').sub(%r{^postgres://}, 'postgis://') %>
```

Just makes it a little more obvious that it's the protocol being replaced.